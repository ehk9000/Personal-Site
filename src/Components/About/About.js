import React from 'react';

const About = () => {
  return (
    <section>
      <article>
        <p>Hi! My name is Erik Konner, and I'm a Finance and Customer Service Associate
           turned Front End Software Engineer.My strengths include React, Redux, problem 
           solving, and communication. I'm an empathetic developer who thrives in teams 
           and loves learning new skills.</p>
        <p>At Colorado College I studied Political Science and Economics, and after 
          graduating I found employment at Oppenheimer Funds. There, after receiving my 
          Series 6 financial license, I worked in their inbound call center, performing 
          trades and was a resource to educate clients on their investments. Life in that
          job presented new puzzles every day, and I found great satisfaction in solving 
          these new problems.</p>
        <p>After a full year working there, I yearned for new skills - something to set me
           apart and that encompassed the challenge and freshness of my old job. That is 
           what led me to Turing and all the excitement of joining a new industry. </p>
      </article>
    </section>

  )

}

export default About;