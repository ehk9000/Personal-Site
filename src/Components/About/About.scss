.about-wrapper {
  display: flex;
  flex-direction: column ;
  min-height: 50vh;
  width: 50vw;
  justify-content: space-between;
  margin: auto;
}

.about-me {
  padding: 15px;
  margin: 20px;
  align-self: center;
  border: 2px solid $primary-color;
  border-radius: 10px;
  p {
    margin-top: 15px;
    text-align: justify;
    line-height: 2.8em;
    font-size: .9rem;
    margin: .7rem auto;
  }
}
.prof-pic {
  height: 319px;
  align-self: center;
  width: 269px;
  border-radius: 50%;
  border: 4px solid #565264;
}



