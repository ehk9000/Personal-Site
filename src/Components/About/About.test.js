import React from 'react';
import { shallow } from 'enzyme';
import About from './About';

describe('About', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<About /> )
  });

  it('Should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});