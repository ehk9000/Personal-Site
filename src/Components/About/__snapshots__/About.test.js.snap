// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Should match snapshot 1`] = `ShallowWrapper {}`;
